ivadomed/scripts/curate_data_demo.py

```diff
@@ -0,0 +1,224 @@
+"""
+The data are structure in a folder such as:
+dataset/
+├── 20220309_demo_mouse_07_0001
+│         ├── image.png
+│         ├── mask.png
+│         ├── mask_seg-axon-manual.png
+│         ├── mask_seg-myelin-manual.png
```

Comment:
@mariehbourget should we go with : 
│         ├── image_seg-axon.png
│         ├── image_seg-axonmyelin.png
│         ├── image_seg-myelin.png
?

Reply:
My apologies for the delay.
If we go we "empty" files, I think it would be more clear for the user that they are labels with the prefix "mask" instead of "image".